hack/jenkins/benchmark-dockerized.sh

```diff
@@ -49,8 +49,12 @@ export GO111MODULE=off
 
 # Install tools we need
 pushd "${KUBE_ROOT}/hack/tools" >/dev/null
-  GO111MODULE=on go install github.com/cespare/prettybench
-  GO111MODULE=on go install gotest.tools/gotestsum
+  hack_tools_gotoolchain="${GOTOOLCHAIN:-}"
+  if [ -n "${KUBE_HACK_TOOLS_GOTOOLCHAIN:-}" ]; then
+    hack_tools_gotoolchain="${KUBE_HACK_TOOLS_GOTOOLCHAIN}";
+  fi
+  GOTOOLCHAIN=$hack_tools_gotoolchain GO111MODULE=on go install github.com/cespare/prettybench
+  GOTOOLCHAIN=$hack_tools_gotoolchain GO111MODULE=on go install gotest.tools/gotestsum
```

Comment:
```suggestion
  GOTOOLCHAIN="${hack_tools_gotoolchain" GO111MODULE=on go install gotest.tools/gotestsum
```
(pretty sure the linter will fail on this otherwise because it could word-split and we clearly want this to be one value assigned to GOTOOLCHAIN, same for the other call sites)

Reply:
(technically the braces are optional for this use case, but the diff for the braces is small and it's easier than trying to ensure they're used when necessary, so I suggest we just use them everywhere, the quotes are the necessary part)

Reply:
quoted and braced everywhere